modules/core/src/main/java/org/apache/synapse/message/store/impl/jms/JmsStore.java

```diff
@@ -119,6 +121,11 @@
 
     private SynapseEnvironment synapseEnvironment;
 
+    /** regex for secure vault expression */
+    private static final String SECURE_VAULT_REGEX = "\\{(wso2:vault-lookup\\('(.*?)'\\))\\}";
+
+    private Pattern vaultLookupPattern = Pattern.compile(SECURE_VAULT_REGEX);
```

Comment:
Can we use the the variables defined in SecureVaultResolver without re-defining them?

Reply:
Fixed